Detectors/TPC/base/include/TPCBase/CDBTypes.h

```diff
@@ -129,13 +134,19 @@ const std::unordered_map<CDBType, const std::string> CDBTypeMap{
   // correction maps
   {CDBType::CalCorrMap, "TPC/Calib/CorrectionMapV2"},
   {CDBType::CalCorrMapRef, "TPC/Calib/CorrectionMapRefV2"},
+  // correction maps for MC
+  {CDBType::CalCorrMapMC, "TPC/Calib/CorrectionMapMCV2"},
+  {CDBType::CalCorrMapRefMC, "TPC/Calib/CorrectionMapRefMCV2"},
```

Comment:
This is a derivative map, right? Perhaps it is better to reflect it in the name, in case in future we have plain reference map for MC.

Reply:
Yes you are right. I will fix it